lib/ohai/dsl/plugin.rb

```diff
@@ -82,6 +83,10 @@ class Plugin
       include Ohai::Mixin::Command
       include Ohai::Mixin::SecondsToHuman
 
+      # Pending next major release of Ohai.
```

Comment:
We should probably decide first if we want it to be removed at all or if FileHelper should just stay as part of the Plugin base API. That would seem fine too :)